tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/PackageExplorer.java

```diff
@@ -136,7 +136,7 @@ public void openClass(ClassContext context) {
     }
 
     public void selectPackage(PackageContext context) {
-        getPackage(context).select();
+        getPackage(context).select().expand();
```

Comment:
mh... this should only select the package, maybe you need another method that also expands that?